workflow-dev/tpl/deis-controller-rc.yaml

```diff
@@ -53,13 +53,13 @@ spec:
                 secretKeyRef:
                   name: builder-key-auth
                   key: builder-key
-{{- if or (eq (env "DATABASE_LOCATION") "off-cluster") (eq .database_location "off-cluster") }}
+{{- if env "DATABASE_LOCATION" | default .database_location | eq "off-cluster" }}
             - name: DEIS_DATABASE_NAME
-              value: "{{ or (env "DATABASE_NAME") .database.name }}"
+              value: "{{ env "DATABASE_NAME" | default .database.name }}"
```

Comment:
This change isn't functional-- it's just cleaner syntax, but it matches the other relevant changes stylistically.